plotly/plotly/plotly.py

```diff
@@ -869,7 +868,7 @@ def _fill_in_response_column_ids(cls, request_columns,
         for req_col in request_columns:
             for resp_col in response_columns:
                 if resp_col['name'] == req_col.name:
-                    req_col.id = '{0}/{1}'.format(grid_id, resp_col['uid'])
+                    req_col.id = '{0}:{1}'.format(grid_id, resp_col['uid'])
```

Comment:
> How is my local grid getting updated if only a url after py.upload() is run?

`py.grid_ops.upload(g)` updates the columns in `g`. Those columns in `g` have the same reference as `c1` and `c2`, so changing them in `g` changes `c1` and c2` too. See http://nedbatchelder.com/text/names.html for some more info.